packages/core/base/core-base-server-mocks/src/core_context.mock.ts

```diff
@@ -23,7 +23,7 @@ function create({
   logger?: jest.Mocked<LoggerFactory>;
   configService?: jest.Mocked<IConfigService>;
 } = {}): DeeplyMockedKeys<CoreContext> {
-  return { coreId: Symbol(), env, logger, configService };
+  return { coreId: Symbol(), env: env as DeeplyMockedKeys<typeof env>, logger, configService };
```

Comment:
It never ocured to me, if we don't need an env mock why would we create one? This is only necessary because env now potentially includes `Package` objects which are not mocked, but `DeeplyMockedKeys<T>` doesn't seem to handle Arrays anyway so I don't think there's an issue, I'm not certain though. This was just a type error that I didn't know what do to about.